src/_generic/features.ts

```diff
@@ -11,5 +13,9 @@ export const getFeature = (apiUrl: string) => (api: ApiInstance) => (id: string)
   api.get(`${apiUrl}/${id}`);
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const createFeature = (apiUrl: string) => (api: ApiInstance) => (body: any): GotPromise<any> =>
-  api.post(apiUrl, { body });
+export const createFeature = (apiUrl: string, mapper: (obj: any) => any = identity) => (api: ApiInstance) => (
```

Comment:
You could create something like:

```typescript
type MapperFn = (obj: any) => any
```

this will generate something more clearly like:
```typescript
export const createFeature = (apiUrl: string, mapper: MapperFn = identity) => (api: ApiInstance) => 
```

Reply:
Done. I had done it in sales/types but forgot to do it here.